Bundle/Bundle-NonWGSScenario5-ProductsofConception-Example.json

```diff
@@ -0,0 +1,922 @@
+    ],
+    "link": [
+     {
+      "other": {
+       "reference": "https://int.api.service.nhs.uk/personal-demographics/FHIR/R4/Patient/9449308322"
+      },
+      "type": "seealso"
+     }
+    ]
+   },
+   "response": {
+    "status": "200 OK",
+    "outcome": {
+     "resourceType": "OperationOutcome",
+     "issue": [
+      {
+       "severity": "information",
+       "code": "informational",
+       "diagnostics": "No issues detected during validation"
+      }
+     ]
+    }
+   }
+  },
+  {
+   "fullUrl": "urn:uuid:037d8ff3-da35-4fad-9d28-7a120d4fffe9",
+   "resource": {
+    "resourceType": "Patient",
+    "id": "037d8ff3-da35-4fad-9d28-7a120d4fffe9",
+    "extension": [
+     {
+      "url": "http://hl7.org/fhir/6.0/StructureDefinition/extension-Patient.fetalStatus",
+      "valueCode": "fetal-demise"
+     }
+    ],
+    "identifier": [
+     {
+      "system": "urn:oid:2.16.840.1.113883.2.1.3.2.4.18.24",
+      "value": "FoetusA",
+      "assigner": {
+       "identifier": {
+        "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+        "value": "RAX"
+       }
+      }
+     }
+    ],
+    "name": [
+     {
+      "family": "Fetus A of MUM"
+     }
+    ],
+    "gender": "female",
+    "link": [
+     {
+      "other": {
+       "reference": "urn:uuid:ac29fd1f-6749-4771-9633-25f39976e48a",
+       "type": "RelatedPerson"
+      },
+      "type": "seealso"
+     }
+    ]
+   },
+   "response": {
+    "status": "200 OK",
+    "outcome": {
+     "resourceType": "OperationOutcome",
+     "issue": [
+      {
+       "severity": "information",
+       "code": "informational",
+       "diagnostics": "No issues detected during validation"
+      }
+     ]
+    }
+   }
+  },
+  {
+   "fullUrl": "urn:uuid:a0684de7-7cf5-43a3-b3ba-3dd13e4c2f4f",
+   "resource": {
+    "resourceType": "Observation",
+    "id": "a0684de7-7cf5-43a3-b3ba-3dd13e4c2f4f",
+    "status": "preliminary",
+    "code": {
+     "coding": [
+      {
+       "system": "http://snomed.info/sct",
+       "code": "723621000000103",
+       "display": "Ethnicity"
+      }
+     ]
+    },
+    "subject": {
+     "reference": "urn:uuid:c60f7e06-79b1-4a34-9979-e8a461b3f7c8",
+     "type": "Patient",
+     "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9449308322"
+     }
+    },
+    "valueString": "unknown"
+   },
+   "response": {
+    "status": "200 OK",
+    "outcome": {
+     "resourceType": "OperationOutcome",
+     "issue": [
+      {
+       "severity": "information",
+       "code": "informational",
+       "diagnostics": "No issues detected during validation"
+      }
+     ]
+    }
+   }
+  },
+  {
+   "fullUrl": "urn:uuid:a5838f88-99af-4ef9-bcb6-150781680d4d",
+   "resource": {
+    "resourceType": "ServiceRequest",
+    "id": "a5838f88-99af-4ef9-bcb6-150781680d4d",
+    "meta": {
+     "tag": [
+      {
+       "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-FundingCategory",
+       "code": "nhs-england",
```

Comment:
Is there any need for this tag?

Reply:
No, this should be within the extensions element of the SR